### PR TITLE
Fix Line interactivity in corner case in MVT

### PR DESCRIPTION
**Pull request by davidmanzanares**
Fix for https://github.com/CartoDB/carto-vl/issues/677

### CODE REVIEW

File: src/client/mvt/feature-decoder.js [hunk 1]
```diff
@@ -266,9 +266,27 @@ function clipLine(line) {
...
+
+    clippedLine = removeDuplicatedVerticesOnLine(clippedLine);
     if (clippedLine.length > 0) {
         clippedLines.push(clippedLine);
     }
 
     return clippedLines;
 }
+
+function removeDuplicatedVerticesOnLine(line) {
```
Comment (elenatorro) [inline src/client/mvt/feature-decoder.js]: Minor thing: private functions start with `_` 😬The rest looks ✨ 👍

Comment (davidmanzanares) [reply src/client/mvt/feature-decoder.js]: Since when? :slightly_smiling_face:

Comment (elenatorro) [reply src/client/mvt/feature-decoder.js]: We have discussed this before many times. Private functions (both a method in a class and a function) starts with `_`. If it is going to be used outside, then it does not.

Comment (davidmanzanares) [reply src/client/mvt/feature-decoder.js]: Ok...

**Pull request merged**